agent/api/errors.go

```diff
@@ -46,7 +46,7 @@ type NamedError interface {
 	ErrorName() string
 }
 
-// NamedError is a wrapper type for 'error' which adds an optional name and provides a symetric marshal/unmarshal
+// NamedError is a wrapper type for 'error' which adds an optional name and provides a symmetric marshal/unmarshal
```

Comment:
Looks like this comment should be moved above to the `NamedError`.

Reply:
I'll change it to talking about `DefaultNamedError`, since that's the implementation that actually provides the marshal/unmarshal behavior.